### PR TITLE
changes in doc structure

### PR DESCRIPTION
**Pull request by bariela**
## Description

Split tutorials in "Use" category for basic functionality and "Advanced" for local development and use of templates.
Added short tutorial/note to starting package also in documentation.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have checked to ensure that there aren't other open [Pull Requests](../../../pulls) for the same update/change.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: doc/tutorials/README.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
+
+- [Quick start with Docker](./quick-start.md) 
+
+## Advanced
+
+More advanced changes, requiring development setup.
+
+- [Run application locally for development] - More detailed version of Getting Started
+- [Run application with sample data](./advanced/run-application-with-sample-data.md) - How to run application with sample data, what kind of sample data do we provide and which functionality is presented with them
+- [Run application with own data](./advanced/run-application-with-own-data.md) - How and where to add another data set into the application
```
Comment (srezacova) [inline doc/tutorials/README.md:16]: dot

File: doc/tutorials/README.md [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
+
+How to use application and make small configuration settings.
+
+- [Quick start with Docker](./quick-start.md) 
+
+## Advanced
+
+More advanced changes, requiring development setup.
+
+- [Run application locally for development] - More detailed version of Getting Started
```
Comment (srezacova) [inline doc/tutorials/README.md:14]: maybe make Getting Started a link

dot

File: doc/tutorials/quick-start.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Quick start
+
+Oscar is web based application produced for cloud enviromnent, therefore it is not optimized for running on own computer. For this reason, you need [Docker](https://www.docker.com/products/docker-desktop) to emulate correct environment. 
+
+Then download [Starting package](../oscar_starting_package.zip) and follow instructions in README.txt in the folder.
+
+This package will download all necessary application parts in Docker and start Oscar with provided Sample data.
```
Comment (srezacova) [inline doc/tutorials/quick-start.md:7]: Here we have Sample data with big S.

File: doc/tutorials/README.md (hunk 1 above)
Comment (bariela) [reply doc/tutorials/README.md:16]: not sure they should be at the end of bullet list items

Comment (srezacova) [reply doc/tutorials/README.md:16]: or we can use small letter in the beginning. We already have dots in the end in main README.md.

**Pull request merged**